### PR TITLE
Store entries when word count submitted

### PR DESCRIPTION
**Pull request by frostyshadows**

### CODE REVIEW

File: src/models/entryModels.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface Entry {
```
Comment (rwblickhan) [inline src/models/entryModels.ts:1]: Can this just be `entry.ts`? (and same for `project.ts`)

File: src/pages/ProjectCreationPage.tsx [hunk 2]
```diff
@@ -43,7 +46,9 @@ export default function ProjectCreationPage() {
       description: description,
     };
     projects.push(newProject);
-    setPersistedProjects(projects);
+    setPersistedProjects(projects).then(() => {
+      navigate(-1);
```
Comment (rwblickhan) [inline src/pages/ProjectCreationPage.tsx]: Can we use `await` syntax here instead?
```javascript
await setPersistedProjects(projects);
navigate(-1);
```

File: src/storage.ts [hunk 3]
```diff
@@ -15,3 +17,26 @@ export async function setPersistedProjects(projects: Project[]) {
     projects: JSON.stringify(projects),
   });
 }
+
+export async function getPersistedEntries(): Promise<Entry[]> {
+  const temp = await chrome.storage.sync.get(null);
```
Comment (rwblickhan) [inline src/storage.ts]: Is this line necessary?

File: src/storage.ts [hunk 4]
```diff
@@ -15,3 +17,26 @@ export async function setPersistedProjects(projects: Project[]) {
...
+}
+
+export async function setPersistedEntries(entries: Entry[]) {
+  await chrome.storage.sync.set({
+    entries: JSON.stringify(entries),
+  });
+}
+
+export async function addPersistedEntry(entry: Entry) {
+  await getPersistedEntries().then((entries) => {
```
Comment (rwblickhan) [inline src/storage.ts]: We don’t need the `then` if we use `await`:
```javascript
const entries = await getPersistedEntries();
entries.push(entry);
setPersistedEntries(entries);
```

**Review by rwblickhan — APPROVED** [review]

File: src/storage.ts (hunk 3 above)
Comment (frostyshadows) [reply src/storage.ts]: I don't think so, I copy and pasted it from the other storage function

File: src/storage.ts (hunk 4 above)
Comment (frostyshadows) [reply src/storage.ts]: lol oops

File: src/models/entryModels.ts (hunk 1 above)
Comment (frostyshadows) [reply src/models/entryModels.ts:1]: I'll keep it like this for now, I like it when file names specify what they're for

**Pull request merged**